README.md

```diff
@@ -136,6 +136,9 @@ convert values beteen JavaScript and C#. Here's a general summary of conversions
   - Custom marshaling and marshaling hints [may be supported later](
     https://github.com/jasongin/napi-dotnet/pull/25).
 
+### Stream across .NET and JS
```

Comment:
>### [](http://example.com/codeflow?start=0&length=3)

nit: could we limit the length of strings?

Reply:
Can you clarify the suggestion? I'm confused because this is about streams, not strings.

Reply:
Fixed.